chdb/driver/driver_test.go

```diff
@@ -0,0 +1,46 @@
+package chdb
+
+import (
+	"database/sql"
+	"testing"
+)
+
+func TestDb(t *testing.T) {
+	db, err := sql.Open("chdb", "")
```

Comment:
Maybe, We could support session with the second arg As path.

Reply:
Yes, I want to support session in the next pr. currently I want to complete SQL interface